### ISSUE TITLE
can't make relation-graph as suggested in Ubuntu 20

### ISSUE THREAD

**Issue by turbomam** [cmungall/semantic-sql#11]
`~/git-repos/semantic-sql$ make bin/relation-graph` 

relation-graph-1.1.tgz gets downloaded and unpacked, but the following `&& (cd bin`... fails

> relation-graph-1.1/bin/relation-graph.bat
/bin/sh: 1: cd: can't cd to bin
make: *** [Makefile:56: bin/relation-graph] Error 2`

**Comment by turbomam** [comment]
Actually I can't even complete the build on a Catalina MacBook

**Comment by turbomam** [comment]
see #12 
will try again after making a `bin` directory

**Comment by turbomam** [comment]
yes, `make bin/relation-graph` does work if a `bin` directory is present

**Comment by turbomam** [comment]
but maybe the lines that invoke relation-graph in the Makefile should use bin/relation-graph

**Comment by turbomam** [comment]
I made the edit but can't push

```
MAM@MAM-M74 semantic-sql % git push --set-upstream origin makefile-bin-relation-graph
ERROR: Permission to cmungall/semantic-sql.git denied to turbomam.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by cmungall** [comment]
remember you can always fork a repo

**Comment by cmungall** [comment]
you have an invite now...

but eventually this should move to an org, after we figure out the optimal arrangement of different functionality
